students/Lincoln_Zhang/Session9/cli_main.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
```

Comment:
It looks like you don't support adding to a donor that already exists yet, instead it overwrites users when trying to add an existing donor.